pages/index.html

```diff
@@ -25,12 +25,12 @@
       </nav>
       <div class="usa-banner-content" id="main-content">
         <h1>U.S. Web Design Standards</h1>
-        <h2 class="usa-font-lead">Open source UI components and a visual style guide for U.S. federal government websites.</h2>
+        <h2 class="usa-font-lead">Open source UI components and visual style guide to create consistency and beautiful user experiences across U.S. federal government websites.</h2>
```

Comment:
Per Writing Lab clarification—phrases that are not complete sentences in this context do not get periods. I will update this PR!